opensrp-anc/src/main/java/org/smartregister/anc/library/util/AncLocationUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.smartregister.anc.library.util;
+
+import android.support.annotation.NonNull;
+
+import java.util.ArrayList;
+
+public class AncLocationUtils {
+
+    @NonNull
+    public static ArrayList<String> getLocationLevels() {
+        ArrayList<String> allLevels = new ArrayList<>();
+        allLevels.add("Country");
+        allLevels.add("Province");
+        allLevels.add("Department");
+        allLevels.add("Health Facility");
+        allLevels.add("Zone");
+        allLevels.add("Residential Area");
+        allLevels.add("Facility");
+        return allLevels;
```

Comment:
This can be defined on the `build.gradle` as build options.  In case another instance uses different ones they will need to extend this. Some instances have already implemented this on the `build.gradle`